flexget/plugins/clients/deluge.py

```diff
@@ -262,6 +302,13 @@ def __init__(self):
                             }
                         },
                         'additionalProperties': False
+                    },
+                    'keys': {
+                        'type': 'array',
+                        'items': {
+                            'type': 'string',
+                            'enum': extra_settings_map.keys()
```

Comment:
This line is failing the tests in py3 for the same reason. Need to convert it to a list because of how py3 changed dict.